doc/configuring.rst

```diff
@@ -12,7 +12,8 @@ file name or file path, use ``--config`` / ``-cfg`` `command-line parameter <arg
 or `CONFIG_PATH` environment variable.
 
 Internally the config file is processed by the :mod:`universum.launcher` module. The path is passed
```

Comment:
please fix :mod:`universum.launcher` here and in its page to refer to the page (`universum.modules.launcher`)